python/nano/test/pytorch/tests/test_inference_pipeline.py

```diff
@@ -0,0 +1,132 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+from torch import nn
+import torch
+from unittest import TestCase
+
+import torchvision.transforms as transforms
+from torchvision.datasets import CIFAR10
+from torch.utils.data import DataLoader
+from torch.utils.data.sampler import SequentialSampler
+from bigdl.nano.pytorch import Trainer
+from bigdl.nano.pytorch.inference import Optimizer
+import torchmetrics
+import torch
+import torch.nn.functional as F
+
+
+data_transform = transforms.Compose([
+        transforms.ToTensor(),
+        transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))
+    ])
+
+
+test_data_transform = transforms.Compose([
+        transforms.ToTensor(),
+        transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))
+    ])
+
+
+def create_data_loader(dir, batch_size, num_workers, transform, subset=50, shuffle=True, sampler=False):
+    train_set = CIFAR10(root=dir, train=True,
+                        download=True, transform=transform)
+    # `subset` is the number of subsets. The larger the number, the smaller the training set.
+    mask = list(range(0, len(train_set), subset))
+    train_subset = torch.utils.data.Subset(train_set, mask)
+    if sampler:
+        sampler_set = SequentialSampler(train_subset)
+        data_loader = DataLoader(train_subset, batch_size=batch_size, shuffle=shuffle,
+                             num_workers=num_workers, sampler=sampler_set)
+    else:
+        data_loader = DataLoader(train_subset, batch_size=batch_size, shuffle=shuffle,
+                             num_workers=num_workers)
+    return data_loader
+
+
+def create_test_data_loader(dir, batch_size, num_workers, transform, subset=50):
+    '''
+    This function is to create a fixed dataset without any randomness
+    '''
+    train_set = CIFAR10(root=dir, train=False,
+                        download=True, transform=transform)
+    # `subset` is the number of subsets. The larger the number, the smaller the training set.
+    mask = list(range(0, len(train_set), subset))
+    train_subset = torch.utils.data.Subset(train_set, mask)
+
+    data_loader = DataLoader(train_subset, batch_size=batch_size, shuffle=False,
+                             num_workers=num_workers)
+    return data_loader
```

Comment:
I think you may sinmply import these utils functions here: https://github.com/intel-analytics/BigDL/blob/8f75766d0e9381e25721be9eacccd19585cbec15/python/nano/test/pytorch/utils/_train_torch_lightning.py

Reply:
Done. thanks!